### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by poema-cavalcanti**
@dlwong @janicelam21 @creativespoon please review.

### CODE REVIEW

File: client/index.jsx [hunk 1]
```diff
@@ -17,4 +17,4 @@ class App extends React.Component {
   }
 }
 
-ReactDOM.render(<App/>, document.getElementById('reservation'));
\ No newline at end of file
+export default App;
```
Comment (creativespoon) [inline client/index.jsx]: Newline at the end

File: test/client/index.test.js [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
-  expect(true).toBe(true);
+import App from '../../client/index';
+
+describe('App component', () => {
+  it('renders', () => {
+    const component = renderer.create(<App></App>);
+    const json = component.toJSON();
+    expect(json).toMatchSnapshot();
+  });
 });
```
Comment (creativespoon) [inline test/client/index.test.js]: Newline at end

File: client/index.jsx (hunk 1 above)
Comment (poema-cavalcanti) [reply client/index.jsx]: Good catch! Thanks!

**Review by creativespoon — APPROVED** [review]
Approved, Snapshot looks like fun!

**Pull request merged**